numpy/core/_add_newdocs.py

```diff
@@ -5655,10 +5655,11 @@ def luf(lamdaexpr, *args, **kwargs):
         number of outputs; use `None` for uninitialized outputs to be
         allocated by the ufunc.
     where : array_like, optional
-        Values of True indicate to calculate the ufunc at that position, values
-        of False indicate to leave the value in the output alone.  Note that if
-        an uninitialized return array is created via the default ``out=None``,
-        then the elements where the values are False will remain uninitialized.
+        The condition is broadcast over the input. If the value is True, output
+        will be set to the ufunc output; otherwise it is left unchanged. Note
+        that if an uninitialized return array is created via the default
+        ``out=None``, output at the position where the values are False will
+        remain uninitialized.
```

Comment:
Better. How about:
```
This condition is broadcast over the input. At locations where the
condition is True, the `out` array will be set to the ufunc result.
Elsewhere, the `out` array will retain its original value.
Note that if an uninitialized `out` array is created via the default
``out=None``, locations within it where the condition is False will
remain uninitialized.
```

With the changes being:

* "condition" helps separate it from the normal "value" arguments, `x1..xn`
* pluralizing things makes this a little clearer.
* Referring to the `out` array explicitly maybe helps emphasize where the uninitialized result comes from.

Perhaps this is just bike-shedding though.

Reply:
adopted

Reply:
.. and synced the other `where` docstring as well